app/workers/historical_data_deletion_worker.rb

```diff
@@ -35,7 +35,7 @@ def delete_and_log(model)
 
   def empty_subscriber_lists
     SubscriberList.left_outer_joins(:subscriptions)
-                  .where("subscriber_lists.created_at < ?", retention_period)
+                  .where("subscriber_lists.created_at < ?", 7.days.ago)
```

Comment:
Could we clarify what the "retention_period" now refers to? I'd suggest moving it and this time period to a couple of constants:

- `MAX_RETENTION_PERIOD`
- `LIST_RETENTION_PERIOD`

Reply:
Good point on the naming clarification!

I've used methods instead of constants due to the fact constants don't change once declared. This is an issue if using dynamic logic like whats being used.

Also changed your suggestion for the list retention to 'empty_list_retention_period` as that is more reflective of what it is.

Reply:
Oops, good point 👍. And thanks for fixing up.